tests/app/platform_stats/test_rest.py

```diff
@@ -228,12 +228,12 @@ def test_volumes_by_service_report(
                            'service_id': str(fixture['service_with_out_ft_billing_this_year'].id),
                            'service_name': fixture['service_with_out_ft_billing_this_year'].name,
                            'sms_chargeable_units': 0, 'sms_notifications': 0}
-    assert response[2] == {'email_totals': 0, 'free_allowance': 10, 'letter_cost': 0.0, 'letter_sheet_totals': 0,
+    assert response[4] == {'email_totals': 0, 'free_allowance': 10, 'letter_cost': 0.0, 'letter_sheet_totals': 0,
```

Comment:
Same comment as above on why we aren't checking rows 2, 3, 5

Reply:
decision: leave a comment in test explaining why we're not checking all rows

Reply:
Dome :)